index.html

```diff
@@ -29,7 +29,6 @@
 
             <h2 class="video-title">Video Title - By Artist Name</h2>
```

Comment:
Review correct indentation using a double spacing bar rather than a tab.

- [ ] correct indentation levels all over the file.